### DISCUSSION TITLE
Get message ID of a quoted message

### DISCUSSION THREAD

**Discussion by juanallendexmarts** [tulir/whatsmeow#616] [WhatsApp protocol Q&A] [answered]
Is it possible to get the message ID (or is it the event id? the one you would get from `event.Info.ID`) from a quoted message (`ExtendedTextMessage.ContextInfo.QuotedMessage`)? If not that, the sender's JID or some other way to identify the original message are useful too. Thanks!

**Comment by ghost** [comment, marked as answer]
`event.Message.<MessageType>.ContextInfo.StanzaID`

https://pkg.go.dev/go.mau.fi/whatsmeow@v0.0.0-20240625083845-6acab596dd8c/proto/waE2E#ContextInfo

**Reply by juanallendexmarts** [reply]
Thanks bro ily
